.github/workflows/deploy-fastly-snippets.yaml

```diff
@@ -13,15 +22,15 @@ jobs:
     env:
       FASTLY_SERVICE_ID: ${{ secrets.FASTLY_SERVICE_ID }}
       FASTLY_API_TOKEN: ${{ secrets.FASTLY_API_TOKEN }}
+      DELETE_SNIPPETS: ${{ github.event.inputs.delete-fastly-snippets }}
+      LOCK_AND_ACTIVATE_SERVICE: ${{ github.event.inputs.lock-and-activate-service }}
     steps:
       - name: Checkout repo
         uses: actions/checkout@v4
       - uses: actions/setup-node@v3
         with:
           node-version: 18
-      - name: Fastly snippet - Redirects table
+      - name: Deploy Fastly snippets
         run: node .github/workflows/deploy-fastly-snippets.js
         env:
```

Comment:
Can we  remove this if no value is specified for env?

Reply:
Nope, I've removed it.